bin/deploy.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 #
-# Usage: ./bin/deploy.sh awslabs-startup-kit-templates-deploy-v2 startup
+# Usage: ./bin/deploy.sh awslabs-startup-kit-templates-deploy-v3 startup
```

Comment:
It seems unfortunate to have to create a new bucket per iteration. Are you trying to maintain backwards compatibility to previous iterations?

Reply:
Totally. I already have an existing ticket open to track this. You have to touch way too many files for each release.